app/basicComponents/Tooltip.js

```diff
@@ -24,27 +30,39 @@ const ToolTipIcon = styled.img`
 const Text = styled.div`
   font-size: 10px;
   line-height: 13px;
-  text-transform: uppercase;
-  color: ${smColors.white};
+  color: ${isDarkModeOn ? smColors.white : smColors.black};
+`;
+
+const OuterIcon = styled.img`
+  width: 13px;
+  height: 13px;
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+  margin-left: 5px;
```

Comment:
make margin-top as class parameter and set it default to 2px

Reply:
fix